bot/exts/evergreen/realpython.py

```diff
@@ -0,0 +1,81 @@
+# Credit to Vthechamp22, who heavily inspired this bit of code
+import logging
+from html import unescape
+from urllib.parse import quote_plus
+
+from discord import Embed, HTTPException
+from discord.ext import commands
+
+from bot import bot
+from bot.constants import Colours
+
+logger = logging.getLogger(__name__)
+
+API_ROOT = "https://realpython.com/search/api/v1/?"
+ARTICLE_URL = "https://realpython.com{article_url}"
+SEARCH_URL = "https://realpython.com/search?q={user_search}"
+
+ERROR_EMBED = Embed(
+    title="Error while searching Real Python",
+    description="There was an error while trying to reach Real Python. Please try again shortly.",
+    color=Colours.soft_red
+)
+
+
+class RealPython(commands.Cog):
+    """User initiated command to search for a Real Python article."""
+
+    def __init__(self, bot: bot.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["rp"])
+    @commands.cooldown(1, 10, commands.cooldowns.BucketType.user)
+    async def realpython(self, ctx: commands.Context, *, user_search: str) -> None:
+        """Sends 5 articles that match the user's search terms."""
+        params = {"q": user_search, "limit": 5}
+        async with self.bot.http_session.get(url=API_ROOT, params=params) as response:
+            if response.status == 200:
+                data = await response.json()
+            else:
+                logger.error(f"Status code is not 200, it is {response.status}")
+                await ctx.send(embed=ERROR_EMBED)
+                return
```

Comment:
```suggestion
            if response.status != 200:
                logger.error(f"Status code is not 200, it is {response.status}")
                await ctx.send(embed=ERROR_EMBED)
                return

            data = await response.json()
```
Is there a benefit structuring it like this?

Reply:
None that are obvious to me, went ahead and changed that.